stdlib/configparser.pyi

```diff
@@ -5,31 +5,53 @@ from re import Pattern
 from typing import Any, ClassVar, TypeVar, overload
 from typing_extensions import Literal, TypeAlias
 
-__all__ = [
-    "NoSectionError",
-    "DuplicateOptionError",
-    "DuplicateSectionError",
-    "NoOptionError",
-    "InterpolationError",
-    "InterpolationDepthError",
-    "InterpolationMissingOptionError",
-    "InterpolationSyntaxError",
-    "ParsingError",
-    "MissingSectionHeaderError",
-    "ConfigParser",
-    "RawConfigParser",
-    "Interpolation",
-    "BasicInterpolation",
-    "ExtendedInterpolation",
-    "LegacyInterpolation",
-    "SectionProxy",
-    "ConverterMapping",
-    "DEFAULTSECT",
-    "MAX_INTERPOLATION_DEPTH",
-]
-
-if sys.version_info < (3, 12):
-    __all__ += ["SafeConfigParser"]
```

Comment:
What was wrong with this syntax? This looks far more readable to me.

Reply:
`__all__` was changed in 3.12 from `list` to `tuple`, I reflected that

Reply:
Yeah, unfortunately stubtest will (reasonably, if you squint at it the right way) complain if `__all__` is a tuple at runtime but a list in the stub

Reply:
Ah, I missed that.